Dockerfile

```diff
@@ -0,0 +1,35 @@
+# To build image:
```

Comment:
We could likely tie these instructions into the makefile with some variables to make it easier. That could be a post merge action -- @akofink thoughts?

Reply:
I think I'm actually opposed to this. Dockerfile's are self-contained and are supposed to "just work" rather than being super configurable like (i.e.) an Ansible playbook. Anyone who's ever used docker should be comfortable with the `docker build` and are free to specify custom args (such as tag). The `docker run` examples are sufficient imo to show different run configurations.

If you really want more configurability, then I'd consider adding a docker-compose yaml before a Makefile.

Reply:
fwiw, I've seen this pattern (dockerfile with comments at the top for run examples) from [reputable container people](https://github.com/jessfraz/dockerfiles/blob/master/chrome/stable/Dockerfile).

Reply:
katello-host-tools does this with a makefile, so we have some precedent.